apps/gitness/src/pages-v2/repo/repo-sidebar.tsx

```diff
@@ -188,21 +188,25 @@ export const RepoSidebar = () => {
 
   return (
     <>
-      <RepoSidebarView
-        hasHeader
-        hasSubHeader
-        selectBranchOrTag={selectBranchOrTag}
-        useRepoBranchesStore={useRepoBranchesStore}
-        useTranslationStore={useTranslationStore}
-        navigateToNewFile={navigateToNewFile}
-        navigateToFile={navigateToFile}
-        filesList={filesList}
-      >
-        {!!repoDetails?.body?.content?.entries?.length && (
-          <Explorer repoDetails={repoDetails?.body} selectedBranch={selectedBranchTag.name} />
-        )}
-      </RepoSidebarView>
-      <Outlet />
+      <div className="grid grid-cols-[auto_1fr]">
+        <RepoSidebarView
+          selectBranchOrTag={selectBranchOrTag}
+          useRepoBranchesStore={useRepoBranchesStore}
+          useTranslationStore={useTranslationStore}
+          navigateToNewFile={navigateToNewFile}
+          navigateToFile={navigateToFile}
+          filesList={filesList}
+        >
+          {!!repoDetails?.body?.content?.entries?.length && (
+            <Explorer repoDetails={repoDetails?.body} selectedBranch={selectedBranchTag.name} />
+          )}
+        </RepoSidebarView>
+        {/* 100vh = screen height - (55px Breadcrumbs Height + 45px SubHeader Height = 100px) */}
+        {/* Total height of both the divs should be 100vh */}
+        <div className="min-h-[calc(100vh-100px)]">
```

Comment:
same style is added on same file in `ui` package as well here: `packages/ui/src/views/repo/repo-sidebar/index.tsx`
why?

Reply:
It seems the file names are confusing. Both of them are needed, one for the actual side bar, and one for the outlet!

https://github.com/user-attachments/assets/d40772b5-e557-4f73-a8ad-eb6671f278ef

Reply:
i see, i think maybe it would be useful to have some comments within the layouts @vivek-harness